storage/raft.go

```diff
@@ -92,9 +93,9 @@ func newSingleNodeRaft(storage multiraft.Storage) *singleNodeRaft {
 	return snr
 }
 
-var _ raftInterface = (*singleNodeRaft)(nil)
+var _ raftInterface = (*raftImpl)(nil)
 
-func (snr *singleNodeRaft) createGroup(id int64) error {
```

Comment:
Yes. This whole file could use a cleanup. We don't need a separate raftInterface if raftImpl is the only implementation of it (and there's no way to use an alternative implementation if there were), and it might be simpler to just move all of its methods out to Store or Range directly.